internal/trace/context.go

```diff
@@ -36,7 +36,7 @@ func NewLogger(ctx context.Context, debug bool) (context.Context, logrus.FieldLo
 	}
 
 	logger := logrus.New()
-	logger.SetFormatter(&logrus.TextFormatter{DisableQuote: true})
+	logger.SetFormatter(&TextFormatter{})
```

Comment:
Is there any UX change if `DisableQuote` is not set any more?

Reply:
Our current implementation does not enforce quoting, and this behavior is equivalent to `DisableQuote`.